cmd/cli/plugin/package/featureflags.go

```diff
@@ -6,7 +6,7 @@ package main
 const (
 	// Package Plugin Kctrl Command Tree determines whether to use the command tree from kctrl. Setting feature flag to
 	// true will allow to use the package command tree from kctrl for package plugin
-	FeatureFlagPackagePluginKctrlCommandTree = "features.package.kctrl-package-command-tree"
```

Comment:
Is it ok to remove the old feature flag ? @anujc25 Would there be any issue if this flag is present in config.yaml

Reply:
Yes. It is fine to remove the old flag. It shouldn't be an issue if the old flag is present.